features/step_definitions/metadata.js

```diff
@@ -17,7 +17,7 @@ module.exports = function server() {
         this.jwt = null;
     });
 
-    this.Given(/^a metadata starts with an empty object$/);
+    this.Given(/^a metadata starts with an empty object$/, { timeout: TIMEOUT }, () => null);
```

Comment:
Hmm.. why we need timeout here? This step does not look like it's doing anything.